python/lsst/afw/geom/polygon/polygon.cc

```diff
@@ -20,67 +20,96 @@
  * see <https://www.lsstcorp.org/LegalNotices/>.
  */
 
+#include <memory>
+
 #include <pybind11/pybind11.h>
-//#include <pybind11/operators.h>
 #include <pybind11/stl.h>
 
+#include "lsst/base.h"  // for PTR
+#include "lsst/pex/exceptions/Runtime.h"
+#include "lsst/pex/exceptions/python/Exception.h"
+#include "lsst/afw/geom/Box.h"
+#include "lsst/afw/geom/Point.h"
+#include "lsst/afw/geom/AffineTransform.h"
+#include "lsst/afw/geom/XYTransform.h"
 #include "lsst/afw/geom/polygon/Polygon.h"
+#include "lsst/afw/table/io/Persistable.h"
+#include "lsst/afw/table/io/pybind11.h"
 
 namespace py = pybind11;
-
 using namespace py::literals;
 
-using namespace lsst::afw::geom::polygon;
+namespace lsst { namespace afw { namespace geom { namespace polygon {
 
 PYBIND11_PLUGIN(_polygon) {
     py::module mod("_polygon", "Python wrapper for afw _polygon library");
 
-    py::class_<Polygon, std::shared_ptr<Polygon>> clsPolygon(mod, "Polygon");
+    // TO DO: Commented-out code is waiting until needed and is untested.
+    // Add tests for it and enable it or remove it before the final pybind11 merge.
```

Comment:
Rename `TODO:` easier to find. But note that I haven't put in such a nice descriptive note in many other places so We will probably end up grepping for `//` anyway, or look at the final diff before merge to master.